### PR TITLE
feat(EntityStatusTag): Added Entity Status Tag component

### PR DESCRIPTION
**Pull request by wichniowski**
This PR will add the EntityStatusTag which simplifies color-coding the entity status.
image

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  .addParameters({
+    propTypes: EntityStatusTag['__docgenInfo'],
+  })
+  .add('default', () => (
+    <EntityStatusTag
+      statusLabel={select(
+        'statusLabel',
+        {
+          'Published (default)': 'published',
+          Archived: 'archived',
```
Comment (mshaaban0) [inline packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx:17]:
```suggestion
          archived: 'archived',
```

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx:17]: @mshaaban0 ahh ja fantastisch wunderbar.. I like the entityStatusType.. will change ... :)

Comment (artibella) [reply packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx:17]: Would adding the `entityStatusType` prop maybe limit the reuse of the Tag component to only appear in the context of some entities that have a content lifecycle? Could I still use the component for non-content entities (e.g. showing the status of a user invite)?

Comment (mshaaban0) [reply packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx]: @artibella you should be able to use the Tag `tagType` prop as it is now `<Tag tagType='warning' />`. the entityStatusType is just an extra prop.

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx]: @artibella I added this as an extra prop so it is configurable like mo described it

Comment (artibella) [reply packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx]: @mshaaban0 @wichniowski Thanks for clarifying!

**Pull request merged**